tests/sandbox/sandbox.sh

```diff
@@ -16,24 +17,34 @@ PLATFORM_FILE="${PATH_HERE}"/../share/PLATFORM
 platform="virtual"
 enclave_type="virtual"
 
-extra_args=("$@")
+extra_args=()
 while [ "$1" != "" ]; do
     case $1 in
         -p|--package)
             is_package_specified=true
+            extra_args+=("$1" "$2")
             shift
             ;;
         -p=*|--package=*)
             is_package_specified=true
+            extra_args+=("$1")
             ;;
         --js-app-bundle)
             is_js_bundle_specified=true
+            extra_args+=("$1" "$2")
             shift
             ;;
         --js-app-bundle=*)
             is_js_bundle_specified=true
+            extra_args+=("$1")
+            ;;
+        -c|--constitution-path)
```

Comment:
Should this be `--constitution-dir`, that expects to find all 4 `*.js` files?

Reply:
Fixed